Add ecosystem_project_resources.rst document (#2007)
ansible/ansible-documentation: docs/docsite/rst/community/ecosystem_project_resources.rst

.. _ecosystem_project_dev_resources:

***********************************************
Ansible Ecosystem Project Development Resources
***********************************************

> This document lists resources that aim to help contributors interested in developing a community project of the `Ansible ecosystem <https://docs.ansible.com/ecosystem.html>`_.

Reviewer: ```suggestion
This document lists resources to help contributors interested in developing a community project in the `Ansible ecosystem <https://docs.ansible.com/ecosystem.html>`_.
```

.. note::

>    Any improvements to the resources listed in this document or to the document itself are very welcome! Please submit a issue or pull request in a corresponding GitHub repository.

Reviewer: ```suggestion
   Any improvements to the resources listed in this document or to the document itself are very welcome! Please submit an issue or pull request in the corresponding GitHub repository.
```

.. _onboarding_toolkit:

Onboarding toolkit
==================

> The `Ansible Onboarding toolkit <https://ansible.readthedocs.io/projects/project-onboarding/en/latest/>`_ provides guidelines on the project GitHub repository setup as well as the type of documentation your project should include.

Reviewer: ```suggestion
The `Ansible Onboarding toolkit <https://ansible.readthedocs.io/projects/project-onboarding/en/latest/>`_ provides guidelines on setting up GitHub repositories for new projects as well as the type of documentation your project should include.
```

> Despite being originally developed for Ansible ecosystem projects under the ansible-community GitHub organization, everyone is welcome to use it.

Reviewer: ```suggestion
Despite being originally developed for Ansible ecosystem projects under the ``ansible-community`` GitHub organization, everyone is welcome to use the onboarding toolkit.
```

Project template
================

> The `Ansible project-template <https://github.com/ansible-community/project-template>`_ is a GitHub repository template for Ansible ecosystem projects. It contains:

Reviewer: ```suggestion
The `Ansible project-template <https://github.com/ansible-community/project-template>`_ is a GitHub repository template for Ansible ecosystem projects that contains:
```

> * files normally present in every repository like README, license- and code-of-conduct-related ones, and others

Reviewer: ```suggestion
* Files normally present in every repository like README, license, code of conduct, and so on.
```

> * Files normally present in every repository like README, license, code of conduct, and so on.

Reviewer 2: ```suggestion
* Files normally present in every repository such as README, license, code of conduct, and so on.
```

> * a docsite template for your project you are encouraged to use to provide a consistent experience to users and contributors across Ansible ecosystem projects.

Reviewer: ```suggestion
* A docsite template that you are encouraged to use with your project to provide a consistent experience across the Ansible project ecosystem.
```

Please take a look at the `docsite <https://ansible.readthedocs.io/projects/ansible-project-template/en/latest/>`_ built from the project-template to see it in action.

> You are welcome to take the template, fill it up with your project-specific information, build it using `Sphinx <https://www.sphinx-doc.org/en/master/>`_ and publish it.

Reviewer: ```suggestion
You are welcome to take the template, fill in your project-specific information, build it using `Sphinx <https://www.sphinx-doc.org/en/master/>`_, and then publish it.
```

> Even if your project is not new and already has documentation, we recommend you take a look at the template and check if there is nothing missed in documentation for your project users, contributors and maintainers.

Reviewer: ```suggestion
Even if your project is not new and already has documentation, we recommend you take a look at the template and check if there is nothing missed in documentation for your project users, contributors, and maintainers.
```

> Even if your project is not new and already has documentation, we recommend you take a look at the template and check if there is nothing missed in documentation for your project users, contributors, and maintainers.

Reviewer 2: ```suggestion
Even if your project is not new and already has documentation, we recommend you take a look at the template and check if there is anything missed in documentation for your project users, contributors, and maintainers.
```

List of community-curated projects
==================================

> Whether your project repository is under the ansible-community GitHub organization or under your own one, you are welcome to include your project in the `Awesome Ansible list <https://github.com/ansible-community/awesome-ansible/blob/main/README.md>`_.

Reviewer: ```suggestion
Whether your project repository is under the ``ansible-community`` GitHub organization or under your own one, you are welcome to include your project in the `Awesome Ansible list <https://github.com/ansible-community/awesome-ansible/blob/main/README.md>`_.
```
